### PR TITLE
Created minishift page and linked from main page

### PR DESCRIPTION
**Pull request by thatdocslady**
This PR handles the new Minishift landing page. Added info about initial install/start based on the Minishift README, as well as links to resources (latest release, docs, community). This is meant to be an overview/welcome page with basic info and links to further info where needed.

Notes:

- I understand that the all-in-one-vm page will be deprecated but I didn't actually delete it yet.
- Updated the link from the main openshift.org page to point to the new minishift page.
- Currently all doc links point to the github live-rendered markdown files. I added comments in the html file as reminders to change the links with the published doc links after we set up the doc publication.

There is a parallel issue for tracking and discussion of this page in the Minishift repo: https://github.com/minishift/minishift/issues/372

### CODE REVIEW

File: source/minishift/index.html.erb
Comment (hferentschik) [inline source/minishift/index.html.erb]: That seems odd to me. You are saying this is a high level summary and that I should refer for more instructions I should refer to the README. At the same time, what follows below is exactly the content of the README. Can we not just point to the README here. At least then we have only one place to maintain and keep the installation instructions up to date. Eventually this might then change to the docs in docs.openshift.org, but for now having the README as single source seems to me to be the best approach.

Comment (hferentschik) [reply source/minishift/index.html.erb]: I just some this up into  single sentence that a hypervisor is needed and one needs to download Minishift. For details see README.

**Review by hferentschik — CHANGES_REQUESTED** [review]
No update to the content of /vm?

File: source/minishift/index.html.erb
Comment (LalatenduMohanty) [inline source/minishift/index.html.erb]: We should also mention that minishift can be run on Microsoft Windows, OS X and GNU/Linux. I know that we have section describing how an user can install in different OSs in the Readme but IMO it would be good to highlight it here.

File: source/minishift/index.html.erb
Comment (LalatenduMohanty) [reply source/minishift/index.html.erb]: Agree with @hferentschik . This way we do not have to change the index page when the Readme gets updated.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: source/minishift/index.html.erb
Comment (thatdocslady) [reply source/minishift/index.html.erb]: Sounds good, I was thinking about doing that initially, not sure what made me go trigger-happy with the copy-paste but I can trim it down.

**Comment by thatdocslady** [comment]
@LalatenduMohanty @hferentschik @thesteve0 please review updated PR with feedback implemented, I cleaned up the get-started section to only mention the hypervisors (it seems to me that it's important but if you want I can also clean it out) and point to the releases page and installation instructions.

BTW if you want to test this page you'll need to install middleman and build it locally, unless @wgordon17 has some magic staging link that he can throw at us? :)

File: source/minishift/index.html.erb
Comment (hferentschik) [inline source/minishift/index.html.erb]: Personally, I find "Local Containerized OpenShift Cluster" a bit misleading. What does "Local" really mean? IMO something like "Develop Applications in a Containerized OpenShift Cluster on your local host". I think this is much less ambiguous.

**Review by hferentschik — APPROVED** [review]
Works for me. Personally I am not a big fan of "Local Containerized OpenShift Cluster". I think "local" is ambiguous in this case. Leave it up to others to decide on this.

**Review by LalatenduMohanty — APPROVED** [review]

**Comment by thatdocslady** [comment]
@thesteve0 all feedback from our side is implemented, please give this PR a pass and approve/merge if OK. Also, if you want me to delete the vm page or make some other adjustment to it as a part of this PR please let me know so we can keep the site clean and tidy :) thanks!

**Comment by thesteve0** [comment]
So if I am reading this correctly, we removed the link to the VM page from the front page and instead pointed it to minishift. If that is the case then I am fine with this - go ahead and merge.
Thanks!

**Comment by thatdocslady** [comment]
@thesteve0 yes we did, but 1- I didn't actually delete the vm page (should I?) and 2- I don't have write access to merge :)

**Comment by thesteve0** [comment]
Do not delete the VM page
I do not have access either - but now they can see we are all on the same page

**Comment by thesteve0** [comment]
@wgordon17 please go ahead and merge

**Pull request merged**
